survey-sdk/src/main/java/co/nimblehq/survey/sdk/request/BaseRequest.kt

```diff
@@ -0,0 +1,20 @@
+package co.nimblehq.survey.sdk.request
+
+import com.squareup.moshi.Json
+
+open class BaseRequest {
+    @Json(name = "client_id")
+    var clientID: String = clientIdInstance
+
+    @Json(name = "client_secret")
+    var clientSecret: String = clientSecretInstance
+
+    companion object {
+        private var clientIdInstance = ""
+        private var clientSecretInstance = ""
+        fun updateKey(id: String, secret: String) {
```

Comment:
need 1 line break between those lines

Reply:
- please provide better param names instead of id and secret
- `updateKey` could be `updateAuthenticationInfo`

Reply:
@luongvo Agree with this